### PR TITLE
Extend semver range to allow React 0.13

### PR DESCRIPTION
**Pull request by terinjokes**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
   ],
   "main": "index.js",
   "dependencies": {
-    "react": "^0.12.1",
+    "react": ">=0.12.1 <0.14.0",
```
Comment (mlmorg) [inline package.json]: Can we make this `>=0.12.0 <0.14.0` to catch `0.12.0` as well?

Comment (terinjokes) [inline package.json]: Done.

**Comment by mlmorg** [comment]
Thanks @terinjokes!

**Comment by mlmorg** [comment]
Published as `v2.1.0`

**Pull request merged**